library/src/main/java/com/mindera/skeletoid/logs/appenders/LogcatAppender.java

```diff
@@ -42,6 +42,9 @@
      * @param tag Log tag
      */
     public LogcatAppender(String tag) {
+        if (tag == null) {
+            throw new IllegalArgumentException("TAG cannot be null");
```

Comment:
I kind of don't like this suggestion, but why not adding the `@NotNull` annotation to the `String tag`, argument.
Also in other places that have this logic.
In part it is better, because that way we don't need to do this kind of verifications at runtime, instead we do it at compile time